libs/plutus-ledger-aeson/plutus-ledger-aeson.cabal

```diff
@@ -40,9 +40,9 @@ library
   -- Other IOG dependencies
   --------------------------
   build-depends:
-    , plutus-core ^>=1.5
-    , plutus-ledger-api ^>=1.5
-    , plutus-tx ^>=1.5
+    , plutus-core ^>=1.15
```

Comment:
So we're not on `>=1.18` yet.

Reply:
Sadly, no https://github.com/IntersectMBO/cardano-api/blob/main/cardano-api/cardano-api.cabal#L203